### PR TITLE
Show the right filter text when clicking a fk link from object view

### PR DESCRIPTION
**Pull request by paulrosenzweig**
I didn't see an issue for this, but I think @walterl flagged this in Slack, so let me know if I missed it.

### Before
before

### After
working

### CODE REVIEW

File: frontend/src/metabase/query_builder/actions.js [hunk 1]
```diff
@@ -1180,9 +1181,10 @@ export const followForeignKey = createThunkAction(FOLLOW_FOREIGN_KEY, fk => {
     const newCard = startNewCard("query", card.dataset_query.database);
 
     newCard.dataset_query.query["source-table"] = fk.origin.table.id;
+    const field = new FieldDimension(fk.origin.id);
     newCard.dataset_query.query.filter = [
       "and",
-      ["=", fk.origin.id, originValue],
+      ["=", field.mbql(), originValue],
```
Comment (paulrosenzweig) [inline frontend/src/metabase/query_builder/actions.js:1187]: I'm not sure how this was working before. I assume it broke in the recent MBQL refactor.

**Review by alxnddr — APPROVED** [review]
👍

**Comment by flamber** [comment]
@paulrosenzweig Pretty sure it's #15079 - unsure if this is the only place with this problem.

**Comment by paulrosenzweig** [comment]
@flamber I think it has the same root cause, but it doesn't seem like normalizing would fix this. The produced MBQL wasn't in the old style it was just a raw ID.

**Pull request merged**